src/Sarif/Writers/ConsoleLogger.cs

```diff
@@ -238,12 +177,18 @@ public static string NormalizeMessage(string message, bool enquote)
 
         public void LogToolNotification(Notification notification)
         {
-            WriteToConsole(notification);
+            if (ShouldLog(notification))
```

Comment:
>ShouldLog [](start = 16, length = 9)

nit: invert condition just like the other places

Reply:
Done.

---
In reply to: [561428877](https://github.com/microsoft/sarif-sdk/pull/2241#discussion_r561428877) [](ancestors = 561428877)